### DISCUSSION TITLE
Preset segment labels

### DISCUSSION THREAD

**Discussion by juliussss** [mifi/lossless-cut#740] [Ideas]
Just a thought - it would be nice to include support for preset/saved segment labels to save on key-presses / re-typing when the same label is reused throughout an edit or across multiple videos.

In my experience I use about 6-10 segment labels consistently and it does get cumbersome when re-typing multiple times. It would also prevent typos which is inevitable with manual keypresses.

Thanks :)

**Comment by mifi** [comment]
Hi! This has been suggested before and is something I want to look into. See #427
